books/free-programming-books-langs.md

```diff
@@ -1603,6 +1603,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Clean Code PHP](https://github.com/jupeter/clean-code-php)
 * [PHP 5 Power Programming](http://www.informit.com/content/images/013147149X/downloads/013147149X_book.pdf) (PDF)
 * [PHP Best Practices](https://phpbestpractices.org)
+* [PHP Documentor](https://docs.phpdoc.org/3.0/)
```

Comment:
ℹ️ Why vrs.3.0? If CONTRIBUTION is read, use latest is out best option.

Link: https://docs.phpdoc.org
Title: `PHP Documentor - Documentation`
Add author `phpdoc.org` to distinguish from other posible resources

Reply:
I get the wrong URL, thanks.